src/components/PackageDocs/index.tsx

```diff
@@ -55,6 +56,8 @@ export function PackageDocs({
   language,
   submodule,
 }: PackageDocsProps) {
+  const [borderColor] = useToken("colors", ["gray.100"]);
+  const border = `1px solid ${borderColor}`;
```

Comment:
What is the need to extract the border color token here?

Can we not just?
```
<Box borderBottom='"1px solid" borderColor="gray.100" justify="center" p={0}>
  <ChooseSubmodule assembly={assembly} />
</Box>
```

Reply:
Good catch, this is a remnant from the previous iteration which incorporated the newer design changes